staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -116,15 +116,15 @@ func NewAuditOptions() *AuditOptions {
 		WebhookOptions: AuditWebhookOptions{
 			BatchOptions: AuditBatchOptions{
 				Mode:        ModeBatch,
-				BatchConfig: defaultLogBatchConfig,
+				BatchConfig: pluginbuffered.NewDefaultBatchConfig(),
```

Comment:
@crassirostris please confirm that these settings are what we want.

Reply:
Yes, though it's preferrable to merge this PR after https://github.com/kubernetes/kubernetes/pull/60926, to avoid showing regressions in the scale tests